src/status/signals/messages.nim

```diff
@@ -174,7 +176,8 @@ proc toChat*(jsonChat: JsonNode): Chat =
     muted: false,
     ensName: "",
     joined: 0,
-    private: jsonChat{"private"}.getBool
+    private: jsonChat{"private"}.getBool,
+    position: jsonChat{"position"}.getInt
```

Comment:
```suggestion
```
`jsonChat` does not have a `position`. It's part of the chat information of the communities. Should be added here:

https://github.com/status-im/status-desktop/blob/master/src/status/signals/messages.nim#L249-L260

Reply:
already remove it just no pushed